### PR TITLE
Account stats panel

### PR DESCRIPTION
**Pull request by dudendy**

### CODE REVIEW

**Review by dmutrokot — APPROVED** [review]

File: src/api/aavegotchiland.api.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { setup } from 'axios-cache-adapter'
+
+const api = setup({
+    baseURL: 'https://api.aavegotchi.land',
+
```
Comment (Vladislav94) [inline src/api/aavegotchiland.api.js:5]: extra line

File: src/components/EthAddressPanel/EthAddressPanel.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    const classes = styles();
+    const [dataLoading, setDataLoading] = useState(true);
+    const [account, setAccount] = useState({});
+
+    useEffect(() => {
+        let mounted = true;
+
+        setDataLoading(true);
+
+        aavegotchilandApi.getAddressInfo(address).then((res) => {
```
Comment (Vladislav94) [inline src/components/EthAddressPanel/EthAddressPanel.js:23]: no need brackets (res)

File: src/components/EthAddressPanel/EthAddressPanel.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                        total: data.gotchies_xp,
+                        inner: [
+                            { name: 'average', value: data.gotchies_medium_xp },
+                        ]
+                    }
+                ];
+
+                setAccount(data.error ? data : formatted);
+            }
+        }).catch((error) => {
```
Comment (Vladislav94) [inline src/components/EthAddressPanel/EthAddressPanel.js:70]: no need brackets (error)

File: src/components/EthAddressPanel/EthAddressPanel.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                        { !account.some(e => e.total > 0) && (
+                            <p className={classes.noStats}>Stats not available for current address :(</p>
+                        )}
+
+                        { account.map((item, i) => {
+                            if (item.total === 0) {
+                                return null;
+                            }
+
+                            return <div className={classes.panel} key={i}>
```
Comment (Vladislav94) [inline src/components/EthAddressPanel/EthAddressPanel.js:119]: maybe wrap in brackets for more clarity

**Pull request merged**